projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -192,6 +196,35 @@ const VideoPressEdit = CoreVideoEdit =>
 			return media;
 		};
 
+		setTracks = async guid => {
+			const { setAttributes } = this.props;
+
+			if ( ! guid ) {
+				return;
+			}
+
+			await apiFetch( {
```

Comment:
Does this throw if the request fails? In async/await the catch portion becomes an exception we should probably handle.